src/main/java/no/nav/tag/sykefravarsstatistikk/api/global/Variable.java

```diff
@@ -0,0 +1,5 @@
+package no.nav.tag.sykefravarsstatistikk.api.global;
+
+public class Variable {
```

Comment:
Fint å trekke ut denne til en global variabel 👍 Jeg foreslår å flytte den til common-folderen vi har, og kanskje kalle den for `Konstanter` i stedet for `Variable`? I og med at det faktisk er en konstant